Plugins/BezierSolution/BezierSpline.cs

```diff
@@ -1048,57 +1051,58 @@ public void AutoConstructSpline()
 				return;
 			}
 
-			Vector3[] rhs;
-			if( m_loop )
-				rhs = new Vector3[n + 1];
-			else
-				rhs = new Vector3[n];
+			int rhsCount = m_loop ? n + 1 : n;
```

Comment:
You can simply rename this to rhsLength and remove rhsLength's original declaration at line 1073.